toptal/picasso
# fix: tooltip position inside of a dropdown

[SPT-1143](https://toptal-core.atlassian.net/browse/SPT-1143)

### Description

For some reason, the tooltip is not showing correctly inside of the dropdown. The tooltip position and tooltip arrow are wrong. 
This PR will fix this issue.

![tooltip_issue](https://user-images.githubusercontent.com/2583281/100196822-1a24ee00-2f02-11eb-88c2-619a683a3c7a.gif)


### How to test

- Navigate here https://picasso.toptal.net/SPT-1143-fix-tooltip-position-issue/?path=/story/overlays--tooltip
- Scroll down, at the end of the page, section Inside of a Dropdown
- Open the dropdown
- Hover the second option
- The tooltip should be visible


### Screenshots

![image](https://user-images.githubusercontent.com/2583281/100197242-ba7b1280-2f02-11eb-9743-b2fd835eda84.png)


### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>
